### ISSUE TITLE
ModuleNotFoundError: No module named 'distutils'

### ISSUE THREAD

**Issue by temuller** [aplpy/aplpy#494]
It seems that starting from Python 12,  distutils is not supported anymore:

Read [What’s New In Python 3.12](https://docs.python.org/dev/whatsnew/3.12.html):

-     PEP 632: Remove the distutils package. See the migration guide for advice replacing the APIs it provided. The third-party Setuptools package continues to provide distutils, if you still require it in Python 3.12 and beyond.

Because of this, I get an error every time I import aplpy. Could you fix this, please?

**Comment by temuller** [comment]
Sorry, I should have described the issue better. I am currently running Github actions to test my code, but I get the following error:

```
/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_local_phot.py:3: in <module>
    from hostphot.cutouts import download_images
/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/hostphot/__init__.py:3: in <module>
    from . import utils
/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/hostphot/utils.py:10: in <module>
    import aplpy
/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/aplpy/__init__.py:14: in <module>
    from .core import FITSFigure  # noqa
/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/aplpy/core.py:1: in <module>
    from distutils import version
E   ModuleNotFoundError: No module named 'distutils'
```

**Comment by astrofrog** [comment]
This should be fixed in APLpy v2.2.0
